source/common/filter/config_discovery_impl.cc

```diff
@@ -17,6 +17,8 @@ namespace Envoy {
 namespace Filter {
 
 constexpr absl::string_view HttpStatPrefix = "http_filter.";
+constexpr absl::string_view TcpListenerStatPrefix = "tcp_listener_filter.";
+constexpr absl::string_view UdpListenerStatPrefix = "udp_listener_filter.";
```

Comment:
nit: I don't think these are used anywhere else, so maybe just inline the return value (even directly in the header file).

Reply:
ACK